aws-ssm-patchbaseline/aws-ssm-patchbaseline.json

```diff
@@ -280,14 +280,19 @@
         "Name"
     ],
     "createOnlyProperties": [
-        "/properties/OperatingSystem"
+        "/properties/OperatingSystem",
+        "/properties/DefaultBaseline",
+        "/properties/Tags"
     ],
     "primaryIdentifier": [
         "/properties/Id"
     ],
     "readOnlyProperties": [
         "/properties/Id"
     ],
+    "writeOnlyProperties": [
```

Comment:
what is WriteOnly, why is OperatingSystem write only?

Reply:
1) "/properties/OperatingSystem" - The writeonly property will help us update the OS details in patchbaseline creation. In the update handler process, we are updating the OS configuration with new updates and its appropriate dependent fields only

Reply:
please explain more. 
OS is not updatable once it's created.

Reply:
readOnlyProperties
Resource properties that can be returned by a read or list request, but cannot be set by the user.

writeOnlyProperties
Resource properties that can be specified by the user, but cannot be returned by a read or list request. Write-only properties are often used to contain passwords, secrets, or other sensitive data.

If any concerns please suggest me.